de.bund.bfr.knime.fsklab.deprecatednodes/src-1_7_2/de/bund/bfr/knime/fsklab/nodes/v1_7_2/joiner/JoinerNodeModel.java

```diff
@@ -336,15 +347,27 @@ private void creatRelationList(String relation, JoinerViewValue joinerProxyValue
           jR.setLanguage_written_in(sourceTargetRelation.getString("language_written_in"));
         }
         if (sourceTargetRelation.containsKey("sourceParam")) {
-          jR.setSourceParam(getObjectFromJson(sourceTargetRelation.get("sourceParam").toString(),
-              Parameter.class));
+        	try {
+        		jR.setSourceParam(getObjectFromJson(sourceTargetRelation.get("sourceParam").toString(),
+        	              Parameter.class));
+        	}catch(Exception e) {
+        		Parameter source =new Parameter();
+            	source.setId(sourceTargetRelation.get("sourceParam").toString());
```

Comment:
Same. You could get the parameter id with getString and then create the parameter:
```java
String parameterId = sourceTargetRelation.getString("sourceParam");
Parameter source = new Parameter().id(parameterId);
```